src/Nethermind/Nethermind.Blockchain/Processing/BlockProcessedEventArgs.cs

```diff
@@ -22,10 +22,12 @@ namespace Nethermind.Blockchain.Processing
     public class BlockProcessedEventArgs : EventArgs
     {
         public Block Block { get; }
+        public TxReceipt[] TxReceipts { get; }
 
-        public BlockProcessedEventArgs(Block block)
+        public BlockProcessedEventArgs(Block block, TxReceipt[] txReceipts)
```

Comment:
should be allowed to be null I guess as we sometimes may not create receipts

Reply:
1. Its not nonnullable context, so it can be null
2. I think we always will create receipts on successful processing, we might not store them though. This comes directly from procesing.